lib/c-api/src/wasm_c_api/externals/global.rs

```diff
@@ -95,16 +95,17 @@ mod tests {
                 wasm_val_t forty_two = WASM_F32_VAL(42);
                 wasm_val_t forty_three = WASM_F32_VAL(43);
 
-                wasm_valtype_t* type = wasm_valtype_new_i32();
-                wasm_globaltype_t* global_type = wasm_globaltype_new(type, WASM_CONST);
+                wasm_valtype_t* valtype = wasm_valtype_new_i32();
+                wasm_globaltype_t* global_type = wasm_globaltype_new(valtype, WASM_CONST);
```

Comment:
Ownership of `valtype` is transferred to `wasm_globaltype_new`, so no need to free it, that's correct.